src/types.jl

```diff
@@ -52,6 +52,7 @@ Base.@kwdef struct DemandFlexibility
     enabled::Bool
     interval_balance::Bool
     rolling_balance::Bool
+    cost_input_type::String
```

Comment:
Could this be renamed to something like `granularity`?

Reply:
I will change it to input_granularity